sdk/ai/Azure.AI.Agents.Persistent/samples/Sample19_PersistentAgents_VectorStoreFile_WithSteps.md

```diff
@@ -0,0 +1,283 @@
+# Sample file search using `VectorStoreFile` and agents in Azure.AI.Agents.Persistent.
+
+In this example we will create the local file, upload it and its file ID to new vector store using. Then we will drill down into run steps agent have ran through.
```

Comment:
I was hard for me to understand this line: "Then we will drill down into run steps agent have ran through". Perhaps a better phrasing will be this short one "Then we will drill down into the Agent's run steps".

Reply:
Fixed.